pkg/controller/nodeipam/ipam/multi_cidr_priority_queue_test.go

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ipam
+
+import (
+	"container/heap"
+	"testing"
+
+	"k8s.io/kubernetes/pkg/controller/nodeipam/ipam/multicidrset"
+	utilnet "k8s.io/utils/net"
+)
+
```

Comment:
We should test the Less() function extensively by itself.

Reply:
Done